.golangci.yaml

```diff
@@ -239,7 +239,6 @@ linters-settings:
       # TODO: enable them all
       disable:
         - bool-compare
-        - compares
```

Comment:
So you removed compare because if this ?

https://github.com/Antonboom/testifylint/issues/93

meaning you wanted testifylint to pass but to use NotSame as it's the right way

Why removing it here instead of adding a `//nolint: testifylint` where you used NotSame?

Reply:
You are right. I read too fast. It's indeed enabled, so my whole question is pointless.

Thanks for replying me anyway